src/pretix/plugins/sendmail/templates/pretixplugins/sendmail/history.html

```diff
@@ -0,0 +1,30 @@
+{% extends "pretixcontrol/event/base.html" %}
+{% load i18n %}
+{% load bootstrap3 %}
+{% block title %}{% trans "Send out emails" %}{% endblock %}
+{% block content %}
+    <h1>{% trans "Email history" %}</h1>
+    <div>
+        <ul class="list-group">
+            {% for log in request.event.all_logentries %}
+                {% if log.action_type == 'sendmail.email.sent' %}
```

Comment:
This will work, but it will be slow for large events, as it first needs to load *all* logs from the databases to drop just a few then. The more elegant way to do this is to already do this in the ``get_queryset`` method and just use ``{% for log in logs %}`` here.